src/App/components/Settings.tsx

```diff
@@ -94,6 +95,7 @@ export default function Settings(): JSX.Element {
   const { settings } = useStores();
   const { analytics } = useAnalytics();
   const repoUrl = repository.url.replace(/\.git$/, '');
+  const resetOutputDir = (): Function => settings.setOutputDirectory(DEFAULT_OUTPUT_DIRECTORY);
```

Comment:
The actual return type of this arrow function should be `void `, i.e. it doesn't return anything (and certainly doesn't return a `Function`).

Reply:
Ok, done